js/common/modules/@arangodb/request.js

```diff
@@ -208,6 +208,7 @@ function request (req) {
 module.exports = request;
 request.request = request;
 request.Response = IncomingResponse;
+module.exports.querystringify = querystringify;
```

Comment:
@dothebart why?

Reply:
FWIW this is a publicly documented module. This exposes an import as an export of this module for no clear reason. If you need the import elsewhere, just import it directly.

Reply:
removed again.